src/actions/index.ts

```diff
@@ -50,46 +48,6 @@ export const setDatabaseConfig = (databaseConfig: DatabaseConfig): SetDatabaseCo
   databaseConfig
 })
 
-export const setExternalSortingUnitMetrics = (a: { sortingId: string, externalUnitMetrics: ExternalSortingUnitMetric[] }): SetExternalSortingUnitMetricsAction => ({
```

Comment:
Removed stuff from this actions/index.ts that are no longer handled in the redux store.